packages/shuttle_http/shuttle_http.0.9.0/opam

```diff
@@ -12,6 +12,7 @@ bug-reports: "https://github.com/anuragsoni/shuttle/issues"
 depends: [
   "dune" {>= "2.8"}
   "shuttle" {= version}
+  "shuttle_ssl" {= version}
```

Comment:
I somehow missed adding a dependency to the ssl library. Since I work in a mono-repo dune picked it up and my ci process didn't catch it since I pin all dependencies in the repo :( 

I hope updating this opam file in the mono-repo is acceptable for now? I'll reproduce this diff upstream. I'm also happy to cut a new release if that'll be better.

Reply:
https://github.com/anuragsoni/shuttle/commit/b38b9c00ec2d367e00693834bd0f9aefeb0cc167 -> fixed upstream

Reply:
Yes, no problem. Opam will use this opam file anyway